ui/src/renderer/ComponentProxy.vue

```diff
@@ -310,11 +311,25 @@ export default {
 			const vnodeProps = {
 				...getRootElProps(),
 			};
-			const renderedComponent = h(template, vnodeProps, {
-				default: defaultSlotFn,
-			});
 
-			return renderedComponent;
+			const parentId = instancePath.at(-2)?.componentId;
```

Comment:
Nice to see you decoded `instancePath`, but no need to flex, I think we can just use `parentId` :)